utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-module-utils",
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
please revert this; package.json should never be bumped in a PR - only in a commit directly made to master.